docs/integrations/databases/opentelemetry/redis-opentelemetry.md

```diff
@@ -70,6 +80,11 @@ OpenTelemetry works with a [configuration](https://opentelemetry.io/docs/collect
 
 In this step, you will configure the yaml file required for Redis Collection.
 
+Below are the inputs required:
+
+- **`endpoint (no default)`**. The hostname and port of the Redis instance, separated by a colon..
```

Comment:
Can you give an example for clarity here

Reply:
fixed it.